test/mailinSpec.js

```diff
@@ -36,7 +55,7 @@ describe('Mailin', function () {
     it('should post a json to a webhook after receiving an email and trigger some events', function (done) {
         this.timeout(30000);
 
-        var doing = 3; // Number of async operations we need to wait for before calling done
+        doing += 4; // Number of async operations we need to wait for before calling done
```

Comment:
Since `doing` is set in `before()`, I have to add to it instead of setting a fixed value.